hypothesis-python/tests/ghostwriter/test_expected_output.py

```diff
@@ -185,6 +190,24 @@ def divide(a: int, b: int) -> float:
                 )
             ],
         ),
+        pytest.param(
+            ("xml_etree_ElementTree_py38", ghostwriter.magic(xml.etree.ElementTree)),
+            marks=[
+                pytest.mark.skipif(
+                    sys.version_info[:2] != (3, 8),
+                    reason="output generated with Python 3.8",
+                )
+            ],
+        ),
+        pytest.param(
+            ("xml_etree_ElementTree_py310", ghostwriter.magic(xml.etree.ElementTree)),
```

Comment:
What's the specific value of these tests?  Can we find an alternative which is consistent between Python versions?

Reply:
Need to think of a class with static/ class methods which is consistent... need to spend more time... I was basically looking from one standard library to another to search for it. Can you think of something?

Reply:
Reading this: https://docs.python.org/3/library/xml.etree.elementtree.html Seems after 3.9 it is stable, I am giving it a try